<launch>
  <arg name="frame_id"        default="GNav" />
  <arg name="child_frame_id"  default="base_link" />
  <arg name="sub_pose_topic"      default="ekf_pose" />
  <arg name="pub_pose_topic"      default="out_pose" />
    <node name="tf_pose" pkg="tf_pose" type="tf_pose.py">
        <param name="frame_id"        value="$(arg frame_id)" />
        <param name="child_frame_id"  value="$(arg child_frame_id)" />
        <param name="sub_pose_topic"      value="$(arg sub_pose_topic)" />
        <param name="pub_pose_topic"      value="$(arg pub_pose_topic)" />
    </node>
</launch>
